bazel_tools/ghc-lib/version.bzl

```diff
@@ -9,7 +9,7 @@ GHC_LIB_PATCHES = [
 ]
 
 GHC_REPO_URL = "https://github.com/digital-asset/ghc"
-GHC_REV = "9cf39eac7128b6653ab697d0ff438fbee5251e05"
+GHC_REV = "4eeb441af413add6e5e7c8721bc08302799bbfee"
```

Comment:
Before merging this PR, please remember to merge https://github.com/digital-asset/ghc/pull/157 and set this to the resulting merge commit, then let CI run once more

Reply:
Ah yes, Dylan mentioned this process :)

Reply:
ah great, just wanted to make sure :)